core/src/components/searchbar/test/basic/index.html

```diff
@@ -98,27 +98,15 @@ <h5 class="ion-padding-start ion-padding-top">Search - Animated and No Cancel</h
         </ion-searchbar>
 
         <h5 class="ion-padding-start ion-padding-top">Search - HTML Placeholder Treated As Text</h5>
-        <ion-searchbar
-          mode="ios"
-          animated="false"
-          show-cancel-button="focus"
-          placeholder="<ion-button>Hello</ion-button>"
-        >
+        <ion-searchbar mode="ios" animated="false" show-cancel-button="focus" placeholder="<button>Hello</button>">
         </ion-searchbar>
 
         <h5 class="ion-padding-start ion-padding-top">Search - Name</h5>
         <ion-searchbar name="search"> </ion-searchbar>
 
-        <div class="ion-padding-horizontal">
-          <ion-button expand="block" onClick="toggleProp()">Toggle Property</ion-button>
-        </div>
-        <div class="ion-padding">
-          <ion-button expand="block" color="secondary" onClick="toggleAttr()">Toggle Attribute</ion-button>
-        </div>
-
-        <div class="ion-padding-horizontal">
-          <ion-button expand="block" color="secondary" onClick="toggleDisabled()">Toggle disabled</ion-button>
-        </div>
+        <button class="expand" onClick="toggleProp()">Toggle Property</button>
+        <button class="expand" color="secondary" onClick="toggleAttr()">Toggle Attribute</button>
```

Comment:
Similar question here around the `color` attribute.

Reply:
Removed in https://github.com/ionic-team/ionic-framework/pull/29422/commits/74d7dd7719335983a82364ba10fb54f9136522ea